### PR TITLE
docs: Update API documentation for Bookings endpoint (_post & _patch)

### PR DESCRIPTION
**Pull request by alishaz-polymath**
## What does this PR do?

Updates the API documentation for Bookings endpoint to correctly consider the Booking Questions and replace the deprecated Custom Inputs from it

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/api/pages/api/bookings/_post.ts [hunk 1]
```diff
@@ -45,29 +41,34 @@ import { defaultResponder } from "@calcom/lib/server";
...
+ *                    - metadata
+ *                    - location
+ *                 properties:
+ *                    name:
+ *                      type: string
+ *                      description: 'Name of the Attendee'
+ *                    email:
+ *                      type: string
+ *                      format: email
+ *                      description: 'Email ID of the Attendee'
```
Comment (emrysal) [inline apps/api/pages/api/bookings/_post.ts:58]:
```suggestion
 *                      description: 'Attendee email address'
```

File: apps/api/pages/api/bookings/_post.ts [hunk 2]
```diff
@@ -45,29 +41,34 @@ import { defaultResponder } from "@calcom/lib/server";
...
+ *                    metadata:
```
Comment (emrysal) [inline apps/api/pages/api/bookings/_post.ts:59]: Is this field required? Maybe it shouldn't be?

**Review by emrysal — APPROVED** [review]
Some nits, otherwise approved

File: apps/api/pages/api/bookings/_post.ts (hunk 2 above)
Comment (alishaz-polymath) [reply apps/api/pages/api/bookings/_post.ts]: This was actually a good catch. I think on my call with @hariombalhara, I must have misinterpreted one of the changes. 🙌 
Resolved now

**Pull request merged**